cypress/cypress/integration/history-links.spec.ts

```diff
@@ -45,6 +46,7 @@ describe("Chat History (Video Mentors Links)", () => {
       gqlQueries: [cyMockGQL("userQuestionSetFeedback", null)],
     });
     cy.visit("/");
+    cy.viewport("macbook-11");
```

Comment:
is the viewport not defined is some `visitDefault` helper function? I prefer not to keep defining the viewport in individual tests